lib/GitCommands.js

```diff
@@ -99,7 +96,10 @@ class GitCommands {
 
     // Create an array based on line breaks, then filter any empty String out of it.
     return output.split(/\r\n|\r|\n/).filter(value => {
-      if (value) return value;
+      if (value) {
+        return value;
+      }
+      return false;
```

Comment:
```suggestion
      return Boolean(value);
```

Github doesn't let me suggest it inline but you can do `filter(Boolean)` too... In most cases using something as a callback that is not intended to be used that way is a bad idea but in the case of `Boolean` I think its fine.